soh/assets/.gitignore

```diff
@@ -4,4 +4,4 @@
 *.cfg
 *.vtx.inc
 *.dlist.inc
-*.txt
```

Comment:
do we know the history of ignoring `.txt` files here? just want to make sure this changes doesn't lead to people accidentally committing extracted assets at some point

Reply:
It's been a while but I don't think I found a reason.